cmd/xgql/main.go

```diff
@@ -94,8 +97,9 @@ func main() {
 		tlsKey   = app.Flag("tls-key", "Path to the TLS key file used to serve TLS connections.").ExistingFile()
 		insecure = app.Flag("listen-insecure", "Address at which to listen for insecure connections.").Default("127.0.0.1:8080").String()
 		play     = app.Flag("enable-playground", "Serve a GraphQL Playground.").Bool()
-		agent    = app.Flag("trace-agent", "Address of the Jaeger trace agent. Leave unset to disable tracing.").String()
+		tracer   = app.Flag("trace-backend", "Tracer to use. Supported backends: \"jaeger\" or \"gcp\"").Default("jaeger").String()
```

Comment:
```suggestion
		tracer   = app.Flag("trace-backend", "Tracer to use.").Default("jaeger").Enum("jaeger", "gcp")
```

I think this will allow us to enforce the two allowed values (and automatically document them for us).